### PR TITLE
updates for doc-2335

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

Updates for the issue DOC-2335.

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployment-with-prometheus.md [hunk 1]
```diff
@@ -68,12 +68,20 @@ In **Continuous Verification Type**, select a type that matches your [deployment
 
 In **Monitored Service**, click **Click to autocreate a monitored service**.
 
-Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. For example, a Service named `todolist` and an Environment named `dev` will result in a Monitored Service named `todolist_dev`.
+Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. For example, a Service named `todolist` and an Environment named `dev` will result in a monitored service with the name `todolist_dev`.
```
Comment (KimberlyFields) [inline docs/continuous-delivery/cd-execution/cv-category/verify-deployment-with-prometheus.md:71]:
```suggestion
Harness automatically creates a monitored service using a concatenation of the service and environment names. For example, a service named `todolist` and an environment named `dev` results in a monitored service with the name `todolist_dev`.
```

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployment-with-prometheus.md [hunk 2]
```diff
@@ -68,12 +68,20 @@ In **Continuous Verification Type**, select a type that matches your [deployment
...
 
-If the stage Service or Environment settings are [Runtime Inputs](../../../platform/20_References/runtime-inputs.md), the Monitored Service and Health Sources settings will show up in the **Runtime Input** settings when you run the Pipeline.
+:::note
+
+The option to auto-create a monitored service is not available if you have configured either service or environment or both as runtime values. When you run the pipeline, Harness concatenates the service and environment values you enter in the runtime inputs screen and generates a monitored service name. If an existing monitored service with the same name exists, Harness assigns it to the pipeline. If no monitored service exists that matches the generated monitored service name, Harness skips the verification step.
```
Comment (KimberlyFields) [inline docs/continuous-delivery/cd-execution/cv-category/verify-deployment-with-prometheus.md:75]:
```suggestion
The option to auto-create a monitored service is not available if you have configured a service, an environment, or both as runtime values. When you run the pipeline, Harness concatenates the service and environment values you enter in the runtime inputs screen and generates a monitored service name. If an existing monitored service with the same name exists, Harness assigns it to the pipeline. If no monitored service that matches the generated monitored service name exists, Harness skips the verification step. 
```

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployment-with-prometheus.md [hunk 3]
```diff
@@ -68,12 +68,20 @@ In **Continuous Verification Type**, select a type that matches your [deployment
...
 
-If the stage Service or Environment settings are [Runtime Inputs](../../../platform/20_References/runtime-inputs.md), the Monitored Service and Health Sources settings will show up in the **Runtime Input** settings when you run the Pipeline.
+:::note
+
+The option to auto-create a monitored service is not available if you have configured either service or environment or both as runtime values. When you run the pipeline, Harness concatenates the service and environment values you enter in the runtime inputs screen and generates a monitored service name. If an existing monitored service with the same name exists, Harness assigns it to the pipeline. If no monitored service exists that matches the generated monitored service name, Harness skips the verification step. 
+
+For example, suppose you enter service as `todolist` and environment as `dev`. In that case, Harness generates the monitored service name `todolist_dev`, checks whether a monitored service with the name `todolist_dev` is available, and assigns it to the pipeline. If no monitored service is available with the name `todolist_dev`, Harness skips the verification step.
```
Comment (KimberlyFields) [inline docs/continuous-delivery/cd-execution/cv-category/verify-deployment-with-prometheus.md:77]:
```suggestion
For example, suppose you enter the service as `todolist` and the environment as `dev`. In that case, Harness generates the monitored service name `todolist_dev`, checks whether a monitored service with the name `todolist_dev` is available, and assigns it to the pipeline. If no monitored service is available with the name `todolist_dev`, Harness skips the verification step.
```

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-app-dynamics.md [hunk 4]
```diff
@@ -63,12 +63,20 @@ In **Continuous Verification Type**, select a type that matches your [deployment
 
 In **Monitored Service**, click **Click to autocreate a monitored service**.
 
-Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. For example, a Service named `todolist` and an Environment named `dev` will result in a Monitored Service named `todolist_dev`.
+Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. For example, a Service named `todolist` and an Environment named `dev` will result in a monitored service with name `todolist_dev`.
```
Comment (KimberlyFields) [inline docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-app-dynamics.md:66]: It looks like the same content is repeated in each file. Apply the same suggestions I added to the previous file to all of the files.

**Review by KimberlyFields — CHANGES_REQUESTED** [review]
I noticed that the yarn.lock file is included in this PR. I'm not sure if you intended for that to be included.

**Comment by SudheendraKatte** [comment]
Created a new branch jira-doc-2335 to address the merge conflict in this PR.

**Pull request closed without merging**